coral/coral/pkg/graphs/resource_models/Consultation.json

```diff
@@ -21,13 +21,13 @@
                     "graph_id": "8d41e49e-a250-11e9-9eab-00224800b26d",
                     "helpenabled": false,
```

Comment:
Take a copy of this so we can use it in the future to replace the current consultation model on develop and then reset this back to how it is on develop